website/project/metadata/osf-open-ended-3.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Open-Ended Registration",
+    "version": 3,
+    "description": "You will be asked to provide a narrative summary of what is contained in your project. There is no minimum character length.",
```

Comment:
Need to remove this as it causes duplication (the question description below is sufficient).